nnvm/python/nnvm/__init__.py

```diff
@@ -10,3 +10,4 @@
 from . import frontend
 
 __version__ = _base.__version__
+print("NNVM is deprecated and will be removed in a future version. Use Relay instead.")
```

Comment:
should we use warnings.warn, https://docs.python.org/3/library/warnings.html? There is a deprecatedwarning

Reply:
@tqchen updated. FutureWarning could be better as it's turned on by default and is for users